packages/account/src/Helpers/utils.ts

```diff
@@ -95,14 +98,14 @@ export const generatePlaceholderText = (selected_doc: string) => {
     }
 };
 
-export const validate =
-    (errors: Record<string, string>, values: Record<string, string>) =>
-    (fn: (value: string) => string, arr: string[], err_msg: string) => {
+export const validate = <T>(errors: Record<string, unknown>, values: T) => {
```

Comment:
```suggestion
export const validate = <T>(errors: Record<string, string>, values: T) => {
```
@yauheni-deriv it might be a string if err_msg is string.. will this work? :)

Reply:
@maryia-deriv this function is being used in many places which will create type errors everywhere as this type should be inferred from values being passed to the function. Right now FormikError<FormikValues> is being assigned which is of type Record<string, unknown>